inc/packages/admin/info.php

```diff
@@ -430,6 +435,70 @@ function add_requirement_notices( ReleaseDocument $release ) : void {
 	}
 }
 
+/**
+ * Render the validation notice.
+ *
+ * Renders the validation status for the package's alias. Also returns a bool
+ * indicating whether the package is "safe" to install - packages which fail
+ * validation are not safe, while those without an alias or with a valid alias
+ * are safe.
+ *
+ * @param DIDDocument $did DID to validate.
+ * @return bool True if the package is "safe" to install, false if install should be blocked.
+ */
+function render_alias_notice( DIDDocument $did ) : bool {
+	$validation = Packages\validate_package_alias( $did );
+	$title = __( 'Domain Alias', 'fair' );
+	$result = false;
+	switch ( gettype( $validation ) ) {
+		case 'string':
+			$message = sprintf(
+				__( '<strong>Validated</strong> as <a href="%s">%s</a>', 'fair' ),
+				esc_url( 'https://' . $validation . '/' ),
+				esc_html( $validation )
+			);
+			$result = true;
+			break;
+
+		case 'NULL':
+			$message = __( 'Not validated: No domain alias is set', 'fair' );
+			$result = true;
+			break;
+
+		default:
+			if ( ! is_wp_error( $validation ) ) {
+				// Invalid type, assume failure.
+				$validation = new WP_Error( 'fair.packages.admin.info.validation_notice.invalid_result', 'An unknown error occurred' );
```

Comment:
`WP_Error` messages elsewhere in the plugin are translated, but I don't think any of those in this PR are translated.